jobbergate-agent/jobbergate_agent/settings.py

```diff
@@ -58,39 +51,21 @@ def compute_extra_settings(cls, values):
         """
         Compute settings values that are based on other settings values.
         """
-        if values.get("SLURM_RESTD_VERSIONED_URL") is None:
-            values["SLURM_RESTD_VERSIONED_URL"] = "{base}/slurm/{version}".format(
-                base=values["BASE_SLURMRESTD_URL"],
-                version=values["SLURM_RESTD_VERSION"],
-            )
+        buzz.require_condition(
+            values["SBATCH_PATH"].is_absolute(),
```

Comment:
It is tricky at the settings level since the error will be raised at importing time for any agent module when the files are not found. With that, test, troubleshoot, and the reuse of components is impossible without the files in place. We can not even mock them on `conftest.py` because the error is triggered before the mock can run.

Considering we check if the files exist downstream when initiating the submission handler, I suggest keeping the settings as they are for now. We may need to refactor the setting to make it possible without affecting maintainability, but that is out of the scope of this task.